libr/core/cconfig.c

```diff
@@ -195,6 +195,11 @@ static void rasm2_list(RCore *core, const char *arch, int fmt) {
 	}
 }
 
+static inline void __setshowrefptr(RConfig *cfg, const char *asmarch, int asmbits) {
+	int show_refptr = (strncmp (asmarch, "x86", 3));
```

Comment:
use bool

Reply:
i would make this config global instead of x86 specific first, otherwise there are many tests to update and we have no room for testing. maybe after release we can evaluate enabling it for some archs, etc